products/opensuse/main.pm

```diff
@@ -654,19 +665,50 @@ sub load_extra_tests () {
     return 0;
 }
 
+sub load_otherDE_tests() {
```

Comment:
Are you sure you don't confuse the otherde with the name gave the
testsuite on my local instance? certainly not the best name.... 
that's unrelated to this function name.
On Fri, 2016-06-10 at 03:53 -0700, Oliver Kurz wrote:
> 
> > +sub load_otherDE_tests() {
> can you rename this to "load_other_de_tests"? I don't mind the upper
> case but in your local verification run I saw that it says
> "otherde_…" and the "otherde" looks weird
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
> 
>